test/libyul/EwasmTranslationTest.cpp

```diff
@@ -102,6 +102,7 @@ string EwasmTranslationTest::interpret()
 	InterpreterState state;
 	state.maxTraceSize = 10000;
 	state.maxSteps = 1000000;
+	state.maxExprNesting = 60;
```

Comment:
Why 60?

Reply:
Good point. Should also be 64 for consistency. Updated

Reply:
There were two locations with 60. Can you check that both are 64 as well?

Reply:
Ah, right. Updated again.